easybuild/easyconfigs/p/Perl/Perl-5.32.0-FCC-21.05.eb

```diff
@@ -0,0 +1,392 @@
+name = 'Perl'
+version = '5.32.0'
+
+homepage = 'https://www.perl.org/'
+description = "Larry Wall's Practical Extraction and Report Language"
+
+toolchain = {'name': 'FCC', 'version': '21.05'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://www.cpan.org/src/%(version_major)s.0']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['efeb1ce1f10824190ad1cadbcccf6fdb8a5d37007d0100d2d9ae5f2b5900c0b4']
+
+builddependencies = [
+    ('groff', '1.22.4'),
+]
+dependencies = [
+    ('zlib', '1.2.11'),  # for Net::SSLeay
+    ('expat', '2.2.9'),  # for XML::Parser
+    ('ncurses', '6.2'),  # for Term::ReadLine::Gnu
+    ('libreadline', '8.0'),  # for Term::ReadLine::Gnu
+    ('DB', '18.1.40'),  # for DB_File
+]
+
+# OpenSSL is required for Net::SSLeay
+osdependencies = [
+    'openssl',
+    ('openssl-devel', 'libssl-dev', 'libopenssl-devel'),
+]
+
+exts_default_options = {'buildopts': 'CC="$CC"'}
+
+
+# !! order of extensions is important !!
+# extensions updated on Aug 21st 2020
+exts_list = [
+    ('Config::General', '2.63', {'source_tmpl': 'Config-General-%(version)s.tar.gz', 'source_urls': ['https://cpan.metacpan.org/authors/id/T/TL/TLINDEN'], 'checksums': ['0a9bf977b8aabe76343e88095d2296c8a422410fd2a05a1901f2b20e2e1f6fad']}),
```

Comment:
not sure what happened in `exts_lists` here (also if `Python` easyconfigs), all extensions are in a single (too long) line, was it `--try-toolchain --copy-ec`?